janusgraph-test/src/main/java/org/janusgraph/graphdb/JanusGraphIndexTest.java

```diff
@@ -1810,7 +1811,22 @@ private void testCollection(Cardinality cardinality, String property, Object val
             assertEquals(v1, getOnlyElement(graph.query().has(property, Geo.WITHIN, Geoshape.circle(2.0, 2.0, 0.1)).vertices()));
         }
 
-
+        
+        // Test traversal property drop Issue #408
```

Comment:
I don't have a strong opinion on this, just wanted to call out as a general anti-pattern from what I've seen elsewhere to cram too many test cases into a single test method, so wanted to suggest that we consider it here. If it does not make sense or would complicate the test cases in a different way, let's keep it as is.